### ISSUE TITLE
FE part that displays did:peer of the wallet with mediator routing for receiving messages

### ISSUE THREAD

**Issue by hugoib** [ADORSYS-GIS/wallet-vc#6]
A FE component that connects with this service https://github.com/ADORSYS-GIS/wallet-vc-libs/blob/main/docs/service-layer/other-events.md#retrieve-didwithmediator

It will retrieve the did:peer where the mediator receives messages and will display it on the wallet for further usage (for example from BOB).

**Comment by IngridPuppet** [comment]
@Awambeng - Isn't this a duplicate of #12 ?

**Comment by Awambeng** [comment]
@IngridPuppet I think #12 is more about the first step of adding the contact to our wallet, while this is related to messaging.

**Comment by nitch2019** [comment]
Waiting for https://github.com/adorsys/eudiw-app/issues/262 to be merged

**Comment by nitch2019** [comment]
Waiting for the library to be fixed. @forkimenjeckayang pls create and add the Ticket here

**Comment by Awambeng** [comment]
Bug has been fixed already

**Comment by Awambeng** [comment]
Waiting for https://github.com/ADORSYS-GIS/Cwallet-vc-libs#77 to be merged

**Comment by Awambeng** [comment]
unblocked can now proceed
